### PR TITLE
Minor: Add example comment on window expr

### PR DESCRIPTION
**Pull request by caicancai**
## Which issue does this PR close?

Add example comment on window expr

### CODE REVIEW

File: datafusion/core/src/dataframe/mod.rs [hunk 1]
```diff
@@ -337,6 +337,20 @@ impl DataFrame {
...
     /// Return a new DataFrame that adds the result of evaluating one or more
     /// window functions ([`Expr::WindowFunction`]) to the existing columns
+    /// # Example
+    /// ```
+    /// # use datafusion::prelude::*;
+    /// # use datafusion::error::Result;
+    /// # #[tokio::main]
+    /// # async fn main() -> Result<()> {
+    /// let ctx = SessionContext::new();
+    /// let df = ctx.read_csv("tests/data/example.csv", CsvReadOptions::new()).await?;
```
Comment (comphead) [inline datafusion/core/src/dataframe/mod.rs:347]: the example might be not complete, would you mind including windows frame?

Comment (caicancai) [reply datafusion/core/src/dataframe/mod.rs:347]: ok

**Comment by comphead** [comment]
Moving to draft as it is expected more actions before PR can be finally reviewed

**Comment by caicancai** [comment]
> Moving to draft as it is expected more actions before PR can be finally reviewed

Thank you, I have been busy with work recently, I will improve this PR next week

**Pull request closed without merging**